lib/middleware/karma.js

```diff
@@ -220,10 +220,10 @@ function createKarmaMiddleware (
           }) : []
 
           return data
-            .replace('%SCRIPTS%', scriptTags.join('\n'))
+            .replace('%SCRIPTS%', () => scriptTags.join('\n'))
             .replace('%CLIENT_CONFIG%', 'window.__karma__.config = ' + JSON.stringify(client) + ';\n')
             .replace('%SCRIPT_URL_ARRAY%', 'window.__karma__.scriptUrls = ' + JSON.stringify(scriptUrls) + ';\n')
```

Comment:
I believe this should be changed to function as well to cover [this branch](https://github.com/karma-runner/karma/blob/master/lib/middleware/karma.js#L107). Please change it and add a corresponding test.